gridcapa-core-cc-post-processing-app/src/main/java/com/farao_community/farao/core_cc_post_processing/app/services/HourlyF303InfoGenerator.java

```diff
@@ -98,10 +95,10 @@ private HourlyF303Info getInfoForNonRequestedOrFailedInterval() {
         return new HourlyF303Info(criticalBranches);
     }
 
-    private HourlyF303Info getInfoForSuccessfulInterval(ProcessFileDto raoResultProcessFile, ProcessFileDto cgmProcessFile) {
+    private HourlyF303Info getInfoForSuccessfulInterval(ProcessFileDto raoResultProcessFile, ProcessFileDto cgmProcessFile, String cracFilename, InputStream cracInputStream) throws IOException {
 
         Network network = getNetworkOfTaskDto(cgmProcessFile);
-        FbConstraintCreationContext cracCreationContext = new FbConstraintCracCreator().createCrac(nativeCrac, network, taskDto.getTimestamp(), cracCreationParameters);
+        FbConstraintCreationContext cracCreationContext = (FbConstraintCreationContext) Crac.readWithContext(cracFilename, cracInputStream, network, taskDto.getTimestamp(), cracCreationParameters);
```

Comment:
to avoid schema version check, you can temporarily replace this with:
```suggestion
        FbConstraintCreationContext cracCreationContext = (FbConstraintCreationContext) new FbConstraintImporter().importData(cracInputStream, cracCreationParameters, network, taskDto.getTimestamp());
```